exercise/staff_views.py

```diff
@@ -338,6 +338,8 @@ def get(self, request: HttpRequest, *args: Any, **kwargs: Any) -> HttpResponse:
             )
             .filter(count_assessed=0))
 
+        total_submissions = submissions.count()
```

Comment:
I think the idea was to show the number of submitters remaining, now the counting is done based on submissions. I made four submissions by three different users and it shows X / 4 (and percentage accordingly), but the number of total students should have been 3.